src/docs/Doxyfile

```diff
@@ -1644,11 +1644,6 @@ ALLEXTERNALS           = NO
 
 EXTERNAL_GROUPS        = YES
 
-# The PERL_PATH should be the absolute path and name of the perl script
```

Comment:
How does this part fit into the change?

Reply:
Not sure how this change got here. I have reverted it.